README.md

```diff
@@ -135,7 +135,7 @@ directories are not allowed.
 ## Integration domain conflict between custom and core integrations
 
 It is possible for a custom integration and a core integration to collide on
-a `domain` name level. In these cases, the core integration domain get
+a `domain` name level. In these cases, the core integration domain gets
```

Comment:
These are unrelated changes to adding new brand assets. Please remove those from this PR and create a separate PR for stuff like this.


Thanks! 👍

../Frenck